audio/sox/files/uint64_t.diff

```diff
@@ -0,0 +1,18 @@
+diff --git src/sox.h src/sox.h
+index 155742e..139e21f 100644
+--- src/sox.h.orig
++++ src/sox.h
```

Comment:
sox.h is a header that gets installed. Since your patch changes this file, the port's `revision` would need to be increased.